### PR TITLE
add converted md files for ws-trust, ws-fed and access control

### PR DESCRIPTION
**Pull request by gomathyk**
## Purpose
add converted md files for ws-trust, ws-fed and access control

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -37,90 +37,67 @@ theme:
...
   
 # Navigation
 nav:
-  - 'Home': index.md
-  - 'Getting Started':
-    - 'Introduction': getting-started/overview.md
+  - 'Introduction':
+    - 'Introduction': introduction/introduction.md
+    - 'Overview': introduction/overview.md
     - 'Architecture':
```
Comment (sherenewso2) [inline en/mkdocs.yml]: Changes such as the 'home' icon and the structure change from "Introduction" to "Getting Started" section need to be there still but have been removed in this PR. Please check on these.

File: en/mkdocs.yml [hunk 2]
```diff
@@ -896,8 +893,6 @@ extra_css:
 extra_javascript:
   - assets/lib/highlightjs/highlight.min.js
   - assets/js/theme.js
-  - assets/lib/backtotop/js/util.js
-  - assets/lib/backtotop/js/main.js
```
Comment (sherenewso2) [inline en/mkdocs.yml]: Check if these .js files should actually be removed?

File: en/mkdocs.yml [hunk 3]
```diff
@@ -849,14 +854,6 @@ nav:
       - 'Supported Cipher Suites': admin-guide/supported-Cipher-Suites.md
       - 'Directory Structure of WSO2 Products': admin-guide/directory-Structure-of-WSO2-Products.md
       - 'WSO2 Patch Application Process': admin-guide/wSO2-Patch-Application-Process.md
-  - 'References': 
-    - 'References': references/references.md
-    - 'Configuration Catalog': references/config_catalog.md
```
Comment (sherenewso2) [inline en/mkdocs.yml]: The configuration catalog needs to be there

File: en/mkdocs.yml (hunk 1 above)
Comment (gomathyk) [reply en/mkdocs.yml]: Added changes to mkdocs.yml.

**Pull request merged**